common/src/main/java/io/druid/metadata/PasswordProvider.java

```diff
@@ -34,5 +34,15 @@
 })
 public interface PasswordProvider
```

Comment:
Point being, changes to `PasswordProvider` should probably be in a PR of their own and not tied up in this one.